rfcs/text/0041/user.yml

```diff
@@ -0,0 +1,170 @@
+---
+- name: user.profile.id
```

Comment:
we don't create `profile` as  separate namespace because it's tuned and should be used only in `user` namespace?